godaddy/metadata.csv

```diff
@@ -1 +1,15 @@
-metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,sample_tags
\ No newline at end of file
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,sample_tags
+godaddy.certificate.canceled_certificates,count,,,,The total number of canceled certificates,0,godaddy,Canceled Certificates,,
+godaddy.certificate.certificate_expiration_days,count,,day,,Count of expiration days by certificate,0,godaddy,Certificate Expiration Days,,
+godaddy.certificate.certificate_status,count,,,,Contains the certificate with status,0,godaddy,Certificate Status,,
+godaddy.certificate.denied_certificates,count,,,,The total number of denied certificates,0,godaddy,Denied Certificates,,
+godaddy.certificate.domain_status,count,,,,The domain with status by certificate,0,godaddy,Domain Status,,
```

Comment:
Same here: "with status" is confusing to me. I'm not entirely sure what you mean?

Reply:
This metric contains the domain name, its current status, and the certificate with which the domain is associated. This metric is also used for monitors. The domain status can be one of the following: (COMPLETED, FAILED_VERIFICATION, VERIFIED_POSSIBLE_FRAUD, DROPPED, DROPPED_CERTIFICATE_AUTHORITY_AUTHORIZATION, DROPPED_GOOGLE_SAFE_BROWSING, INVALID, AWAITING, PENDING_POSSIBLE_FRAUD, REVOKED_CERTIFICATE)